src/test/java/duke/task/DeadlineTest.java

```diff
@@ -0,0 +1,26 @@
+package duke.task;
+import org.junit.jupiter.api.Test;
```

Comment:
Should you leave a newline after the package? Just pointing it out incase you missed it.